scripts/igraph_impl.py

```diff
@@ -0,0 +1,51 @@
+def overall_reciprocity(G):
+    return G.reciprocity()
+
+
+def pagerank(
+    G,
+    alpha=0.85,
+    personalization=None,
+    max_iter=100,
+    tol=1e-06,
+    nstart=None,
+    weight="weight",
+    dangling=None,
+    *,
+    vertices=None,
+    directed=True,
+    arpack_options=None,
+    implementation="prpack",
+):
+    if personalization is not None:
+        raise NotImplementedError
```

Comment:
Personalization is provided through the `reset` and `reset_vertices` parameters of `personalized_pagerank()`

https://python.igraph.org/en/stable/api/igraph.GraphBase.html#personalized_pagerank